packages/riipen-ui/src/components/Paper.jsx

```diff
@@ -0,0 +1,111 @@
+import clsx from "clsx";
+import PropTypes from "prop-types";
+import React from "react";
+import _JSXStyle from "styled-jsx/style";
+
+import ThemeContext from "../styles/ThemeContext";
+import withClasses from "../utils/withClasses";
+
+const Paper = props => {
+  const {
+    children,
+    classes,
+    component: Component,
+    rounded,
+    elevation,
+    variant,
+    ...other
+  } = props;
+
+  const theme = React.useContext(ThemeContext);
+
+  const className=clsx(
+    "root",
+    classes,
+    rounded ? "rounded" : null,
+    variant === "outlined" ? "outlined" : "elevation"
+  );
+  
+  return (
+    <React.Fragment>
+    <Component 
+    className={className}
+      {...other}
+    >
+      {children}
+      </Component>
+     
+      <style jsx>
+        {`
+          .root {
+            background-color: ${theme.palette.common.white};
+            color: ${theme.palette.text.primary}
+            padding: ${theme.spacing(3)}px;
```

Comment:
Now that I see this, I actually think this should be a prop. I would maybe add:

```js
spacing: PropTypes.number,
```
to the props, then apply it here. Maybe the default prop value can be 3 as well, but I would be more tempted to move it to 2 because Riipen uses padding 2 a lot.

Reply:
I'm tempted to call the prop `padding` to be in line with the TableCell component. Grid uses `spacing` as a prop, but it has a slightly different meaning.

Reply:
Sounds good to me :).